Disaster Recovery documentation (#124)
ministryofjustice/staff-device-dns-dhcp-infrastructure: documentation/disaster-recovery.md

# Disaster Recovery

Unplanned downtime needs to be addressed as soon as possible to minimise disruption of end user devices.

A number of potential scenarios, their configured alarms and associated remediation steps are detailed below.

1. [Corrupt configuration file](#corrupt-configuration-file)
2. [Corrupt container was published](#corrupt-container-was-published)

> 3. [Misconfigured infrastructure](bad-infrastructure-apply-with-terraform)

Author: Dead link here

> 4. [DHCP Subnet gets full](#dhcp-subnet-gets-full)

Author: Dead link

5. [Services Overloaded](#services-overloaded)
6. [AWS Availability Zone goes down](#aws-availability-zone-goes-down)

> 7. [Other AWS failures](#other-aws-failures)

Author: Need to delete this heading

# Disaster Recovery

> Any unplanned downtime needs to be addresses as quickly as possible to minimise disruption of the end user devices.

Reviewer: ```suggestion
Unplanned downtime needs to be addressed as soon as possible to minimise disruption of end user devices.
```

> Below is a list of potential causes, how these errors will be raised as alarms and how to remediate them:

Reviewer: ```suggestion
A number of potential scenarios, how these errors will be notified as alarms and remediation steps are detailed below:
```
